metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -128,7 +128,8 @@ object ServerCommands {
         |   mainClass: "com.foo.App",
         |   buildTarget: "foo",
         |   args: ["bar"],
-        |   jvmOptions: ["-Dfile.encoding=UTF-16"]
+        |   jvmOptions: ["-Dfile.encoding=UTF-16"],
+        |   env: ["NUM=123"]
```

Comment:
Ah, I may have misunderstood the VS Code PR. So it's filled in as Map, but pass in as an Array of `key=value` right?

Reply:
My bad! I we do send it as map, but it's changed to list when sent via bsp. Need to fix this.

Reply:
Fixed!